devices.js

```diff
@@ -814,6 +830,15 @@ const devices = [
         fromZigbee: [fz.xiaomi_battery_3v, fz.SJCGQ11LM_water_leak_iaszone],
         toZigbee: [],
     },
+    {
+        zigbeeModel: ['lumi.flood.agl02'],
+        model: 'SJCGQ12LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara water leak sensor',
+        supports: 'water leak true/false',
+        fromZigbee: [fz.xiaomi_battery_3v, fz.SJCGQ12LM_water_leak_iaszone],
```

Comment:
Please try with `fz.ias_water_leak_alarm_1` instead of `fz. SJCGQ12LM_water_leak_iaszone`

Reply:
tried fz.ias_water_leak_alarm_1 instead of fz. SJCGQ12LM_water_leak_iaszone  works perfectly.